govcd/tar.go

```diff
@@ -0,0 +1,106 @@
+package govcd
+
+import (
+	"archive/tar"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func Unpack(tarFile string) ([]string, error) {
```

Comment:
Is this meant to be part of the public API?  If not the name should be lower case.

Reply:
Well it are different code files "catalog" and "util/tar"